test/In.ProjectEKA.HipServiceTest/Link/LinkPatientTest.cs

```diff
@@ -164,12 +169,16 @@ private async void ReturnOtpExpired()
             var otpToken = TestBuilders.Faker().Random.Number().ToString();
             var testOtpMessage = new OtpMessage(ResponseType.OtpExpired, "Otp Expired");
             var patientLinkRequest = new LinkConfirmationRequest(otpToken, sessionId);
+            var linkEnquires = new LinkEnquires("","","ncg","",
+                DateTime.Now.ToString(),new List<CareContext>());
```

Comment:
Codacy found an issue: [Define the locale to be used in this string operation.](https://app.codacy.com/gh/ProjectEKA/hip-service/pullRequest?prid=5773356)